gadgets/chaos_packet_drop/program.bpf.c

```diff
@@ -0,0 +1,385 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2022 Microsoft Corporation
+
+#include <vmlinux.h>
+
+/* I am directly including the 
+value of the constants instead of 
+the linux/if_ether.h header file and linux/pkt_cls.h
+because of redeclaration conflicts with
+vmlinux.h */
+
+#define ETH_P_IP 0x0800 /* Internet Protocol packet	*/
+#define ETH_P_IPV6 0x86DD /* IPv6 over bluebook		*/
+
+#define TC_ACT_UNSPEC (-1)
+#define TC_ACT_OK 0
+#define TC_ACT_RECLASSIFY 1
+#define TC_ACT_SHOT 2
+#define TC_ACT_PIPE 3
+#define TC_ACT_STOLEN 4
+#define TC_ACT_QUEUED 5
+#define TC_ACT_REPEAT 6
+#define TC_ACT_REDIRECT 7
+
+#include <stdbool.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_endian.h>
+#include <gadget/types.h>
+#include <gadget/macros.h>
+#include <gadget/buffer.h>
+
+#define GADGET_TYPE_NETWORKING
+#include <gadget/sockets-map.h>
+
+/* The maximum number of items in maps */
+#define MAX_ENTRIES 8192
+#define TASK_COMM_LEN 16
+
+struct events_map_key {
+	struct gadget_l4endpoint_t dst;
+};
+
+struct event {
+	gadget_timestamp timestamp_raw;
```

Comment:
What's the purpose of having a timestamp here? This gadget provides a "summary" of how many events have been dropped, I can't see how a timestamp is useful.